include/wabt/binary.h

```diff
@@ -21,6 +21,8 @@
 
 #define WABT_BINARY_MAGIC 0x6d736100
 #define WABT_BINARY_VERSION 1
+#define WABT_MODULE_LAYER 0
+#define WABT_COMPONENT_LAYER 1
```

Comment:
These should probably have `WASM_BINARY_` prefix like everything else here.

How about `WASM_BINARY_LAYER_MODULE` and `WASM_BINARY_LAYER_COMPONENT`?

Reply:
Done.